ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml

```diff
@@ -104,6 +106,45 @@ SettingsPageLayout {
                 }
             }
 
+            CommunityMembersTabPanel {
+                model: root.pendingRequestsMembersModel
+                placeholderText: {
+                    if (root.pendingRequestsMembersModel.count === 0) {
+                        return qsTr("No pending requests to search")
+                    } else {
+                        return qsTr("Search %1's %2 pending request%3").arg(root.communityName)
+                                                              .arg(root.pendingRequestsMembersModel.count)
+                                                              .arg(root.pendingRequestsMembersModel.count > 1 ? "s" : "")
+                    }
+                }
+                panelType: CommunityMembersTabPanel.TabType.PendingRequests
+
+                Layout.fillWidth: true
+                Layout.fillHeight: true
```

Comment:
I tend to put these at the beginning of components. I am not sure if it's covered in our qml coding guidelines so I do not insist :D